packages/roc-package-web-app-react/src/config/roc.config.js

```diff
@@ -26,6 +26,7 @@ export default {
                 style: [],
             },
             ssr: true,
+            redirectStatusCode: 301,
```

Comment:
Can we add this to the `roc.config.meta.js` as well with validation and a description?